snuba/datasets/events.py

```diff
@@ -200,6 +213,23 @@ def get_promoted_context_columns(self):
     def get_required_columns(self):
         return self.__required_columns
 
+    def column_expr(self, column_name, body):
```

Comment:
Please define this method in the parent class as well, the code that invokes it will not specific to the event dataset thus it would break this way if the author of a new dataset forgets to create this method.